tests/common/e2e_test.go

```diff
@@ -51,6 +51,10 @@ func e2eClusterTestCases() []testCase {
 			name:   "ClientAutoTLS",
 			config: config.ClusterConfig{ClusterSize: 1, ClientTLS: config.AutoTLS},
 		},
+		{
+			name:   "ClusterWithLearner",
+			config: config.ClusterConfig{ClusterSize: 3, ClientTLS: config.AutoTLS, WithLearner: true},
```

Comment:
```suggestion
			name:   "Learner",
			config: config.ClusterConfig{ClusterSize: 1, WithLearner: true},
```

No need to have 3 member cluster to test learner nor configure client TLS.